src/sentry/static/sentry/app/components/events/interfaces/threads/threadsSelectorSelectedOption.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {Frame} from 'app/types/events';
+import TextOverflow from 'app/components/textOverflow';
+/* eslint-disable react/prop-types */
+import Text from 'app/components/text';
+import space from 'app/styles/space';
+
+const NOT_FOUND_FRAME = '<unknown>';
+
+interface Props {
+  id: string;
+  details: ThreadInfo;
+}
+
+interface ThreadInfo {
+  label: string;
+  filename?: string;
+}
+
+const ThreadsSelectorSelectedOption: React.FC<Props> = ({id, details}) => (
+  <StyledContainer>
+    <StyledThreadID>{`Thread #${id}:`}</StyledThreadID>
+    <StyledOptionLabel>{details.label}</StyledOptionLabel>
+  </StyledContainer>
+);
+
+export default ThreadsSelectorSelectedOption;
+
+const StyledContainer = styled('div')({
+  gridTemplateColumns: '110px 1fr',
+  display: 'grid',
+  justifyContent: 'flex-start',
+  alignItems: 'center',
+  justifyItems: 'start',
+});
+
+const StyledThreadID = styled(TextOverflow)({
```

Comment:
I think we follow the camel case even w/ acronyms so that it's more consistent

```suggestion
const StyledThreadId = styled(TextOverflow)({
```